nbs/tsdataset.ipynb

```diff
@@ -54,13 +54,15 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.                data = pd.read_parquet(self.files_ds.files[idx], columns=self.files_ds.temporal_cols.to_list()).to_numpy()
</pre><p>The columns argument can be an index, no need to call <code>to_list</code> on it </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I've now changed the logic here as we need to sort the dataframes in case the files within the directory aren't read in the right order